Tests/SwiftDocCTests/Test Bundles/TestBundleWithDupMD.docc/documentation/overview.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started with Capybara
+
+Create a capybara and assign personality traits and abilities.
+
+## Overview at documentation level
+
+Capybara are complex creatures that require careful creation and a suitable habitat. After creating a sloth, you're responsible for feeding them, providing fulfilling activities, and giving them opportunities to exercise and rest. 
```

Comment:
I approve of this title, Caplin too :+1: 🤣

```suggestion
Capybara are complex creatures that require careful creation and a suitable habitat. After creating a capybara, you're responsible for feeding them, providing fulfilling activities, and giving them opportunities to exercise and rest. 
```

Reply:
Thank you @ktoso :D